vllm/engine/arg_utils.py

```diff
@@ -1035,6 +1036,12 @@ def create_engine_config(self,
                 "BitsAndBytes load format and QLoRA adapter only support "
                 f"'bitsandbytes' quantization, but got {self.quantization}")
 
+        if envs.VLLM_ATTENTION_BACKEND in [STR_DUAL_CHUNK_FLASH_ATTN_VAL]:
+            assert self.enforce_eager, (
+                "Cuda graph is not supported with DualChunkFlashAttention. "
+                "To run the model in eager mode, set 'enforce_eager=True' "
+                "or use '--enforce-eager' in the CLI.")
```

Comment:
Do you know what the blockers are for Cuda graphs + DualChunkFlashAttention?

Reply:
The current implementation doesn't support cuda graph for two reasons: (1) cuda graph is designed for short sequence length (due to block tables padding), the default capture sequence length is 8192, and, (2) there are some CPU logic (about sequence length) inside the DualChunkFlashAttention backend.